install.sh

```diff
@@ -56,6 +56,26 @@ if [[ $response = 1 ]]; then
   success "Copied Nimble Templates successfully"
 fi
 
+# ----------------------------------
+# UITest Templates
+# ----------------------------------
+section "UITest Templates"
+confirm "Do you want to copy UITest Templates?"
```

Comment:
Please make a suggestion for developers (e.g `1/0` or `yes/no`)

Reply:
This is taken care by Mark's helper 
<img width="448" alt="Screen Shot 2021-07-06 at 15 59 46" src="https://user-images.githubusercontent.com/6356137/124572899-39f69a00-de73-11eb-8510-d2d63ae9c1ba.png">